docker/pasta-skeleton.dockerfile

```diff
@@ -42,7 +42,10 @@ RUN apt install -y --no-install-recommends \
 RUN pip3 --no-cache-dir install \
 	dateparser \
 	flask-bootstrap \
-	flask-nav anytree
+	flask-nav anytree \
+	flat-table \
+	pandas "dask[complete]" \
+	matplotlib seaborn
```

Comment:
Don't we get those libraries via Ubuntu's official repository? I would prefer that over pip for stability reasons.